src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -35,7 +35,7 @@ def prepare_create(self, daemon_spec: CephadmDaemonSpec[IscsiServiceSpec]) -> Ce
             'entity': self.get_auth_entity(igw_id),
             'caps': ['mon', 'profile rbd, '
                             'allow command "osd blocklist", '
-                            'allow command "config-key get" with "key" prefix "iscsi/"',
+                            f'allow command "config-key get" with "key" prefix "iscsi/"',
```

Comment:
opps I can remove the 'f' here. I was going to make the prefix cap match what ever the user put in. But currently the ceph-iscsi ssl certs look for iscsi/ so we need it here. I might go and push the ssl certs into the ceph-iscsi config file so we can loose this requirement.